web/modules/custom/mymodule/src/Form/FirstForm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+class FirstForm extends FormBase {
+  
+  function getFormId() {
+    return 'custom_form_id';
+  }
+
+  function buildForm(array $form, FormStateInterface $form_state) {
+
+    $form['full_name'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Full Name')
+    ];
+    $form['phone_no'] = [
+      '#type' => 'tel',
+      '#title' => $this->t('Phone Number')
+    ];
+
+    $form['email'] = [
+      '#type' => 'email' ,
+      '#title' => $this->t("Email")
+    ];
+
+    $form['gender'] = [
+      '#type' => 'radios',
+      '#title' => $this->t('Gender'),
+      '#options' => array('Male'=>('Male'),'Female'=>('Female'),'Others'=> ('Others'))
+    ];
+    
+    $form['submit'] = [
+      '#type' => 'submit',
+      '#value' => $this->t('Submit'),
+    ];
+
+    return $form;
+  }
+
+  function validateForm(array &$form, FormStateInterface $form_state) {
+    $publicDomains = [];
```

Comment:
All local variables should be camelcase.
Only class variables should be SnakeCased.

Reply:
This is still not fixed.

Reply:
Updated.